packages/orbit-components/src/Stack/Stack.stories.tsx

```diff
@@ -37,845 +38,355 @@ import ChoiceGroup from "../ChoiceGroup";
+      <InputField type="password" label="Password" help="You need some help!" />
+      <Button>Sign In</Button>
+    </Stack>
+  ),
 
-LargeDesktopProperties.argTypes = {
-  direction: {
-    options: Object.values(DIRECTIONS),
-    control: {
-      type: "select",
-    },
-  },
-  align: {
-    options: Object.values(ALIGNS),
-    control: {
-      type: "select",
-    },
-  },
-  justify: {
-    options: Object.values(JUSTIFY),
-    control: {
-      type: "select",
-    },
-  },
-  spacing: {
-    options: Object.values(SPACINGS),
-    control: {
-      type: "select",
-    },
-  },
-  spaceAfter: {
-    options: Object.values(SPACINGS_AFTER),
-    control: {
-      type: "select",
-    },
+  args: {
+    spacing: SPACINGS.EIGHT_HUNDRED,
   },
 };
 
-export const NestedExample = () => (
-  <Stack spacing="600" direction="column" align="start">
-    <Stack spacing="200" direction="row" inline>
-      <Text weight="bold">email@gmail.com</Text>
-      <Edit size="small" color="success" />
-    </Stack>
-    <Stack direction="row" spaceAfter="large" align="end">
-      <InputField type="password" label="Password" error="You need some help!" />
-      <Button>Sign In</Button>
+export const NestedExample: Story = {
+  render: () => (
+    <Stack spacing="600" direction="column" align="start">
+      <Stack spacing="200" direction="row" inline>
+        <Text weight="bold">email@gmail.com</Text>
+        <Edit size="small" color="success" />
+      </Stack>
+      <Stack direction="row" spaceAfter="large" align="end">
+        <InputField type="password" label="Password" error="You need some help!" />
+        <Button>Sign In</Button>
+      </Stack>
+      <Text size="small">
+        <TextLink type="secondary">Forgot password?</TextLink>
+      </Text>
     </Stack>
-    <Text size="small">
-      <TextLink type="secondary">Forgot password?</TextLink>
-    </Text>
-  </Stack>
-);
-
-NestedExample.story = {
-  name: "Nested example",
+  ),
 
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-export const Playground = ({
-  dataTest,
-  flex,
-  inline,
-  direction,
-  wrap,
-  grow,
-  shrink,
-  basis,
-  align,
-  justify,
-  spacing,
-  spaceAfter,
-  legacy,
-  mediumMobile,
-  largeMobile,
-  tablet,
-  desktop,
-  largeDesktop,
-}) => {
-  return (
-    <Stack
-      dataTest={dataTest}
-      flex={flex}
-      direction={direction}
-      align={align}
-      justify={justify}
-      wrap={wrap}
-      grow={grow}
-      basis={basis}
-      inline={inline}
-      shrink={shrink}
-      spacing={spacing}
-      legacy={legacy}
-      spaceAfter={spaceAfter}
-      mediumMobile={mediumMobile}
-      largeMobile={largeMobile}
-      tablet={tablet}
-      desktop={desktop}
-      largeDesktop={largeDesktop}
-    >
+export const Playground: Story = {
```

Comment:
`id` is also missing here.
also, `as` args is not included either, I am considering if we would like to included this arg to our controls (if I remember correctly, we didn't include for some component, bcs there was not visible change (Button, I guess (??))

Reply:
Yeah, I guess we can include it 🤔 ✅